OracleHTTPServer/README.md

```diff
@@ -95,6 +60,31 @@ _This volume will be created in "/var/lib/docker" directory or the location wher
 
 ######Later you can login to running container and configure Weblogic Server proxy plugin file and run *restartOHS.sh* script.
```

Comment:
How do  I handle the restart case? For stopping, I have set a handler for SIGTERM, for restart which signal I should target?